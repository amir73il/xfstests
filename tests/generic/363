#! /bin/bash
# SPDX-License-Identifier: GPL-2.0
# Copyright (c) 2024 Red Hat, Inc.  All Rights Reserved.
#
# FSQA Test No. 363
#
# Run fsx with EOF pollution enabled. This provides test coverage for partial
# EOF page/block zeroing for operations that change file size.
#

. ./common/preamble
_begin_fstest rw auto

_require_test

# currently only xfs performs enough zeroing to satisfy fsx
_supported_fs xfs

# on failure, replace -q with -d to see post-eof writes in the dump output
run_fsx "-q -S 0 -e 1 -N 100000"

status=0
exit
