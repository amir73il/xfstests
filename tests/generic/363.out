QA output created by 363
fsx -q -S 0 -e 1 -N 100000
